x-pack/docs/en/security/operator-privileges/operator-only-snapshot-and-restore.asciidoc

```diff
@@ -0,0 +1,19 @@
+[role="xpack"]
+[testenv="enterprise"]
+[[operator-only-snapshot-and-restore]]
+=== Operator Privileges and Snapshot/Restore
+The invocation of <<operator-only-apis>> and changes to <<operator-only-dynamic-cluster-settings>>
+result changes in the cluster state, which can be <<snapshot-restore,snapshot and restored>>.
+Restoring snapshot data associated to <<operator-only-functionality,operator-only functionality>>
+is problematic because:
+
+1. A snapshot could contain wrong values for operator-only functionalities. Restore them
+   could break the cluster.
+2. Even if we could correct the values immediately after the restore, there will always be
+   a short time period where cluster could be in an inconsistent state.
+3. We prefer configure operator-only functionality from a single place, i.e. through API calls.
+
+Therefore, <<configure-operator-privileges,*when the feature of operator privileges is enabled*>>,
+snapshot data associated to any operator-only functionality are *not* restored.
```

Comment:
This is not yet true for autoscaling policies, which currently do not get restored regardless whether operator privileges are enabled or not. However, I do not want to go into this difference here because this will be the ultimate behaviour that we want once Cloud enables operator privileges.